web/js/date/timeline-input.js

```diff
@@ -283,10 +407,10 @@ export function timelineInput(models, config, ui) {
       if ((selectedDateObj > tl.data.start()) &&
         (selectedDateObj <= util.today())) {
         var parent = selected.parent();
-        var sib = parent.next('div.input-wrapper.selectable')
```

Comment:
I am not able to tab from the day field back around to the year field. This is due to removing the `selectable` class logic here on this line (and the line below) + the toggleSubdaily function is no longer applying those classes (here: https://github.com/nasa-gibs/worldview/blob/fix-year-jump-3/web/js/layers/active.js#L428-L444)